docs/django/first-steps-with-django.rst

```diff
@@ -92,7 +92,10 @@ The uppercase name-space means that all Celery configuration options
 must be specified in uppercase instead of lowercase, and start with
 ``CELERY_``, so for example the :setting:`task_always_eager` setting
 becomes ``CELERY_TASK_ALWAYS_EAGER``, and the :setting:`broker_url`
-setting becomes ``CELERY_BROKER_URL``.
+setting becomes ``CELERY_BROKER_URL``. This also applies to the
+workers settings, and overrides the usual ``CELERYD_`` prefix.
+For instance, the :setting:`worker_concurrency` setting becomes
+``CELERY_CONCURRENCY``.
```

Comment:
Actually, did I get this right? Shouldn't it instead be: 

`the :setting:`worker_concurrency` setting becomes ``CELERY_WORKER_CONCURRENCY``

Reply:
could you check locally?

Reply:
Sure, checking now

Reply:
Just tested and confirmed, will send another PR to fix this page. I think I got confused with the old setting names.